CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## [v1.0.0-rc.4] - 2019-12-17
+
+### Changes
+- Update kooper to v0.8.0
+- Update kubernetes to v1.15.6
+- Add support for `hostNetwork` and `dnsPolicy` in Redis and Sentinel pods
+
+### Fix
+- connect the redis cluster from outside #109 @pmquang
```

Comment:
I think this is not a fix, it would be a new feature but for me, there is no even feature that specifically does this so i would not include it in the changelog.

Reply:
Ok, I added it only because it closed the issue :)